packages/block-editor/src/components/inserter/search-results.js

```diff
@@ -31,6 +33,25 @@ const INITIAL_INSERTER_RESULTS = 9;
  */
 const EMPTY_ARRAY = [];
 
+const orderInitialBlockItems = ( items, priority ) => {
+	if ( ! priority ) {
+		return items;
+	}
+
+	// Sort is "in place".
```

Comment:
What does this comment mean?

Reply:
Haha. Sorry yes. I should have removed that. It was to remind me that `.sort()` will _mutate_ rather than return a new reference like things like `.filter()` would do.